lib/reducers/call-taker.js

```diff
@@ -103,18 +103,20 @@ function createCallTakerReducer () {
         })
       }
       case 'ROUTING_RESPONSE': {
+        const {searchId} = action.payload
         if (state.activeCall) {
           // If call is in progress, record search ID when a routing response is
           // fulfilled.
           // TODO: How should we handle routing errors.
           return update(state, {
-            activeCall: { searches: { $push: [action.payload.searchId] } }
+            activeCall: { searches: { $push: [searchId] } }
           })
-        } else if (state.callHistory.visible) {
+        } else if (state.callHistory.visible && searchId.indexOf('_CALL') === -1) {
```

Comment:
Yes, I'm not entirely sure how to fix this.

Reply:
This may be worse:
1. Start from scratch
2. Open call taker windows and expand an entry
3. Plan a trip on the map, indenpently from the call window.
4. The call button state changes to 'recording'.

![Screen Recording 2021-04-27 at 5 24 11 PM](https://user-images.githubusercontent.com/56846598/116315566-b50a4880-a77e-11eb-8fe2-5bdbd9786465.gif)

Reply:
I submitted #362 to prevent a call from starting when clicking entries in the call log. It should not affect anything that is not dealing with the `_CALL` suffix in search ids. (Note: #362 does NOT address the issue in the animation above.)